core/session.ts

```diff
@@ -153,7 +159,7 @@ export class Session {
     if (action.proto.target) {
       action.proto.target = this.target(
         actionOptions.sqlxConfig.name,
-        actionOptions.sqlxConfig.schema
+        schema
```

Comment:
not sure what this change is for, can you clarify for me?

Reply:
Some other dbadapters have these line to handle assertTasks: 
const target =
      assertion.target ||
      dataform.Target.create({
        schema: projectConfig.assertionSchema,
        name: assertion.name
      });
however, assertion.target always had the default "dataform" schema, not the desired "dataform_assertions"